transformations/transformations/single_column_coalesced.py

```diff
@@ -640,7 +640,12 @@ def process_driver(self, routine, targets=None):
                             loop._update(pragma_post=(ir.Pragma(keyword='acc', content='end parallel loop'),
                                                       loop.pragma_post[0]))
 
-                # Apply hoisting of temporary "column arrays"
+        # Apply hoisting of temporary "column arrays"
+        with pragmas_attached(routine, ir.Loop, attach_pragma_post=True):
```

Comment:
What's the reason for attaching pragmas here? Can't seem to find where this is used.

Reply:
Good point, the pragmas handled in the call to `hoist_temporary_column_arrays` are actually not loop-attached ones, so this is probably redundant. Will remove, thanks.

Reply:
Thanks, looks good now!